src/webots/nodes/WbCadShape.cpp

```diff
@@ -285,8 +285,10 @@ QStringList WbCadShape::objMaterialList(const QString &url) const {
       if (!cleanLine.startsWith("mtllib"))
         continue;
 
-      cleanLine = cleanLine.replace("mtllib", "");
-      materials << cleanLine.split(' ', Qt::SkipEmptyParts);
+      cleanLine = cleanLine.replace("mtllib ", "");
```

Comment:
I would feel safer with this. If the line contains an extra space at the end, that would break the rest of the code.
```suggestion
      cleanLine = cleanLine.replace("mtllib ", "").trimmed();
```